react/components/Slider.js

```diff
@@ -1,7 +1,393 @@
 import React from 'react'
+import React, { Children, Component, cloneElement } from 'react'
+import debounce from 'debounce'
+import PropTypes from 'prop-types'
+import { resolveSlidesNumber } from '../utils'
 
-const Slider = () => (
-  <div>Slider works</div>
-)
+class Slider extends Component {
+  static propTypes = {
+    resizeDebounce: PropTypes.number,
+    onChangeCurrentSlide: PropTypes.func.isRequired,
+    duration: PropTypes.number,
+    easing: PropTypes.string,
+    perPage: PropTypes.oneOfType([PropTypes.number, PropTypes.object]),
+    draggable: PropTypes.bool,
+    threshold: PropTypes.number,
+    howManySlides: PropTypes.number,
+    loop: PropTypes.bool,
+    children: PropTypes.oneOfType([
+      PropTypes.element,
```

Comment:
shouldn't this be a `PropTypes.node`? I know this won't make much sense, but I see it as a perfectly valid use-case, which is to have a slide of strings, it wouldn't pass this restriction.

Reply:
It actually should be
```javascript
PropTypes.arrayOf(PropTypes.instanceOf(Slide))
```